### DISCUSSION TITLE
List of CRDs required for Strimzi Kafka deployment in GKE

### DISCUSSION THREAD

**Discussion by git-azeez** [strimzi/strimzi-kafka-operator#8920] [Q&A] [answered]
Hi Team,

Can somene share the list of CRDs so that I can ask the  GKE cluster admin to whitelist them

**Comment by scholzj** [comment, marked as answer]
You can download and check the installation files to get the list I guess?

**Reply by git-azeez** [reply]
Thank you

**Reply by git-azeez** [reply]
Thank you
